### ISSUE TITLE
How i can build this software?

### ISSUE THREAD

**Issue by y-ponomarev** [zagrodzki/goscope#44]

**Comment by zagrodzki** [comment]
To use the software in its current shape you need to have a prepared Go development environment (see https://golang.org/doc/install). You'll need libusb too. We've only tried in on Linux so far, but I expect it would also work on a Mac without much hassle.

Once you have all prerequisites, build the binary through "go build github.com/zagrodzki/goscope/gui/dummy_screen".

**Comment by y-ponomarev** [comment]
Thank you.

**Comment by zagrodzki** [comment]
we plant to add a README with instructions in some near future (@dtkaczyk). I'll keep this thread open until that happens.

**Comment by stapelberg** [comment]
I think this is the best command to use to download the software + dependencies and install the binary to $(go env GOPATH)/bin:

```
go get -u github.com/zagrodzki/goscope/gui/dummy_screen
```

@zagrodzki The resulting binary is named “dummy_screen”, which is a somewhat unfortunately generic name :). Maybe the main package should be moved to a more descriptive directory. Convention is to have it in `cmd/<something>`

**Comment by zagrodzki** [comment]
Yeah, it started as a dummy demo that demonstrated how to build a GUI app. I'll change the name.
